docs/installation.rst

```diff
@@ -306,6 +322,14 @@ To stop the containers::
 
     docker-compose stop
 
+If you have made changes to the .env or /etc/hosts file, it's a good idea to run::
+
+    docker-compose down
```

Comment:
per @peterbe's point, can we change 'down' to 'stop'?

Reply:
changed to 'stop' 👍